packages/private/docker/index.ts

```diff
@@ -48,12 +48,15 @@ export function buildDockerImage({
       stdio: 'inherit'
     }
   )
-}
 
-export function publishDockerImage(options: DockerImageOptions) {
-  const { name, version } = options
-  buildDockerImage(options)
+  if (
+    !process.env.CI ||
+    !process.env.CIRCLE_BRANCH ||
+    process.env.CIRCLE_BRANCH !== 'master'
+  )
```

Comment:
does this mean only CI can publish the docker images? I think you had this different before, that you could publish stuff manually as well, what was the reasoning behind the change?

Reply:
Good point. I changed this up so that we don't have two different jobs on CircleCI building the Docker Image. But yeah, it should be able to depoy images manually, too, gonna check that.

Reply:
Solved. Should work now by setting environment variable `DEPLOY` to true, e.g. `DEPLOY=true yarn build:docker`